corehq/apps/app_manager/suite_xml/post_process/endpoints.py

```diff
@@ -102,11 +104,19 @@ def get_child(child_id):
             else:
                 arguments.append(Argument(id=arg_id))
 
-        return SessionEndpoint(
-            id=endpoint_id,
-            arguments=arguments,
-            stack=stack,
-        )
+        if respect_relevancy:
```

Comment:
Are you adding this if statement so  `respect_relevancy` does not show up in the XML when it is false?

Reply:
I don't think there is. I was looking in the xml models if there is a flag for that but I could not find one.